staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -110,6 +138,10 @@ type byPriority struct {
 func (a byPriority) Len() int      { return len(a.specs) }
 func (a byPriority) Swap(i, j int) { a.specs[i], a.specs[j] = a.specs[j], a.specs[i] }
 func (a byPriority) Less(i, j int) bool {
+	// All local specs will come first
+	if a.specs[i].apiService.Spec.Service == nil {
```

Comment:
is this how we operate now in the aggegator? So the priorities in https://github.com/sttts/kubernetes/blob/43b8715d82a713bb2bff5dc53d5e213a42d9d115/cmd/kube-apiserver/app/aggregator.go#L170 do not really play role in comparison to external api services?